tests/test_special_ops.py

```diff
@@ -229,11 +230,12 @@ def test_embedding(EmbeddingSize, Batch, M, N, padding_idx, scale_grad_by_freq,
     gems_assert_close(res_in_grad, ref_in_grad, dtype)
 
 
+@pytest.mark.skip("triton_musa unsupport")
```

Comment:
please verify this. resolve_neg is marked as supported in the weekly report.

Reply:
ok, I've confirmed that this case should be enable, we will fix it.